<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuEditor</class>
 <widget class="QWidget" name="MenuEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="MenuEditorTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="InputConfigurationTab">
      <attribute name="title">
       <string>Input Configuration</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="InputSignalLabel">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="MappedNavigationActionLabel">
         <property name="text">
          <string>Command</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="inputSignalUpLabel">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="inputSignalDownLabel">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="inputSignalLeftLabel">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="inputSignalRightLabel">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="inputSignalSetLabel">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="inputSignalClearLabel">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="inputSignalUpActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="inputSignalDownActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="inputSignalLeftActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="inputSignalRightActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="inputSignalSetActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="inputSignalClearActionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MenuDesignTab">
      <attribute name="title">
       <string>Menu Designer</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <item>
        <layout class="QVBoxLayout" name="menuBrowserLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Menu Hierarchy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="menuBrowserTreeView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="menuBrowserButtonLayout">
           <item>
            <widget class="QPushButton" name="addMenuItemButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Add Menu Item</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeMenuItemButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveMenuItemUpButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveMenuItemDownButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Move Down</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="menuWidgetsLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Menu Item Widgets</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="menuItemWidgetsListView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="menuWidgetsButtonLayout">
           <item>
            <widget class="QPushButton" name="addWidgetButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add Widget</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeWidgetButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Preview</string>
         </property>
         <layout class="QGridLayout" name="_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="menuDesignPreviewLayout"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
